pet-owners/app/views/owners/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<a href="/owners/new">New Owner</a>
+<p>
+    <h3>Pet Owners:</h3>
+</p>
+<% @owners.each do |o| %>
+    <div>
+        <p>
+          <a href="/owners/delete?id=<%= o.id %>">Delete</a>
```

Comment:
Have you seen Rails url helpers yet in class? https://guides.rubyonrails.org/routing.html#path-and-url-helpers and https://guides.rubyonrails.org/routing.html#path-and-url-helpers

You could do something like `<%= link_to 'Delete', owners_delete_path(o.id) %>` and Rails would generate the html and the correct url for the link using your routes.